src/memoryPractice/memoryPractice.service.ts

```diff
@@ -16,6 +25,9 @@ export class MemoryPracticeService {
     private memoryPracticeQuestionRepository: Repository<MemoryPracticeQuestion>,
     @InjectRepository(MultipleChoiceQuestion)
     private multipleChoiceQuestionRepository: Repository<MultipleChoiceQuestion>,
+    @InjectRepository(MemoryPracticeAnswer)
+        private memoryPracticeAnswerRepository: Repository<MemoryPracticeAnswer>,
+        private userService: UserService
```

Comment:
Can u put private userService: UserService on top? (Before any @InjectRepository(..)). Got a bug with this before for some reason